openshift/openshift.go

```diff
@@ -88,9 +93,12 @@ func ExportAsTemplate(filter *ResourceFilter, exportOptions *cli.ExportOptions)
 			)
 		}
 		item.RemoveUnmanagedAnnotations()
+		item.removeClusterFields()
 		itemPointer, _ := gojsonpointer.NewJsonPointer("/objects/" + strconv.Itoa(k))
 		itemPointer.Set(m, item.Config)
 	}
+	kindPointer, _ := gojsonpointer.NewJsonPointer("/kind")
+	kindPointer.Set(m, "Template")
```

Comment:
Maybe we can move this to where you exchange `items` for `objects`, I think that and changing the `kind` belongs logically together.